web/crux/prisma/schema.prisma

```diff
@@ -27,12 +27,19 @@ model Team {
   notifications Notification[]
 }
 
+enum UserInvitationStatusEnum {
+  pending
```

Comment:
We don't want to register the `verified` statuses?

Reply:
that's comes from kratos